### DISCUSSION TITLE
Reporting bugs.

### DISCUSSION THREAD

**Discussion by pereshaped** [plugdata-team/plugdata#172] [General]
Whats the procedure for reporting bugs? Sorry to be the bearer of bad news.

Todays release crashes on every session, first locks, then quits. It happens after connecting random objects. Any
Did audio test fine. 

I am using an old 32/64 bit computer system so maybe that is it. But I havent had any issuses with vanilla.

High Sierra
Macbook Air 2012
Running PD Vanilla 0.52.1 very stable.
Latest Else libs, stable too

**Comment by timothyschoen** [comment]
You can report a bug on the "issues" page. And that is bad news indeed! Are you using the standalone version or the plugin version? 

Unfortunately I have nothing that's running High Sierra, so if it's caused by that, it'll be hard for me to replicate. Do you get any kind of crash report?

**Comment by pereshaped** [comment]
I basically start PlugData application straight from my applications folder. Select "New Project" and start populating with patch.

This morning seems to be behaving better. Im doing some work this afternoon on a big vanilla patch so will test again and see how i behaves and if it does quit I'll get a report. Thanks

**Comment by pereshaped** [comment]
Installed on Mac Pro 6,1 High Sierra and so far (3 hours) it seems ok and stable.

**Reply by timothyschoen** [reply]
I'm curious about what was going on earlier, glad it works now though!
